lib/client/api.go

```diff
@@ -4171,6 +4171,18 @@ func (tc *TeleportClient) Ping(ctx context.Context) (*webclient.PingResponse, er
 			Future versions of tsh will fail when incompatible versions are detected.
 			`, pr.MinClientVersion, teleport.Version, pr.MinClientVersion)
 		}
+
+		// Recent `tsh mfa` changes require at least Teleport v15.
+		const minServerVersion = "15.0.0"
```

Comment:
Will this allow 15.0.0-dev and 15.0.0-alpha? I think we want it to but I don't think it currently does

Reply:
Good call, done.